packages/dd-trace/test/telemetry/logs/log-collector.spec.js

```diff
@@ -49,6 +49,12 @@ describe('telemetry log collector', () => {
       ).to.be.false
     })
 
+    it('should not store logs with transmit flag set to false', () => {
+      expect(logCollector.add({ message: 'Error 1', level: 'ERROR' })).to.be.true
+      expect(logCollector.add({ message: 'Error 2', level: 'ERROR', transmit: true })).to.be.true
+      expect(logCollector.add({ message: 'Error 3', level: 'ERROR', transmit: false })).to.be.false
```

Comment:
If I'm correct about my question about `transmit` vs `sendViaTelemetry` above, this test is not actually testing what you think it is. In that case it would be more robust to not just set the flag manually like you do here, but to at least go through the `Log` constructor.